applications/h264/h264_endoscopy_tool_tracking/cpp/h264_endoscopy_tool_tracking.yaml

```diff
@@ -21,7 +21,7 @@ extensions:
   - libgxf_videodecoderio.so
   - libgxf_videoencoder.so
   - libgxf_videoencoderio.so
-  - ../../../gxf_extensions/lstm_tensor_rt_inference/libgxf_lstm_tensor_rt_inference.so
+  - ../../../../gxf_extensions/lstm_tensor_rt_inference/libgxf_lstm_tensor_rt_inference.so
```

Comment:
can remove `libgxf_cuda.so` and `libgxf_multimedia.so` (lines 18-19) above as those extensions are part of GXF and get loaded by default